app/assets/stylesheets/maps.css.scss

```diff
@@ -1,5 +1,4 @@
 /*
- *= require seiyria-bootstrap-slider/dist/css/bootstrap-slider.min.css
```

Comment:
Hmm, i think we might need this for the export resolution setting, can we test this out on a container before merging? 

![image](https://user-images.githubusercontent.com/24359/54782469-60865d80-4bf5-11e9-86e6-7540bc4ab211.png)